src/components/Console.jsx

```diff
@@ -1,11 +1,12 @@
 import React, {useCallback, useEffect, useReducer, useRef, useState} from "react";
 import {listen} from "@tauri-apps/api/event";
-import {CALLBACKS_TRIGGERS, CALLBACK_TYPES} from "./simulator.js";
+import {CALLBACKS_TRIGGERS, CALLBACK_TYPES, STATE} from "./simulator.js";
 import {RUST} from "../rust_functions.js";
+import { debounce } from "lodash";
 
 const ENTER = 13;
 
-function Console({loaded, registerCallback, exitCode, error}) {
+function Console({loaded, registerCallback, exitCode, error, state, stepBack, setState}) {
```

Comment:
console should not need to access step back

Reply:
@WyattRoss this one's you